app/.helper/package

```diff
@@ -213,6 +213,12 @@ install_package() {
     rm -rf "$install_dir/cli/$package_name"
     cp -R  "$pkg_cache_dir"/* "$install_dir/cli"
 
+    local post_install_script="$install_dir/cli/$package_name/scripts/post-install.sh"
+
+    if [[ -f "$post_install_script" ]]; then
+      bash $post_install_script
```

Comment:
This will break the expected output of `install_package()` (`"true"` or `"false"`) if the `$post_install_script` script outputs something to stdout. Maybe you can move the post-install execution to another function called by the caller of `install_package()`?

Reply:
ok.

Reply:
@putrasattvika done